plugins/in_splunk/splunk_conn.c

```diff
@@ -27,26 +27,57 @@
 static void splunk_conn_request_init(struct mk_http_session *session,
                                      struct mk_http_request *request);
 
+static void splunk_conn_session_init(struct mk_http_session *session,
+                                   struct mk_server *server,
+                                   int client_fd);
+
+static int splunk_conn_buffer_realloc(struct flb_splunk *ctx,
```

Comment:
do we not have any unit tests for this?

Reply:
we don't, I suggest internally how this can be added to the CI:

```
[ikslabdein@toolbox repro]$ ./main -payload=payload.json -rps=1 -token=AAAA-AAAA-AAAA-AAAA -endpoint=http://localhost:9393/services/collecto
```